master/Makefile

```diff
@@ -46,13 +46,13 @@ check-gen: force-gen gen
 
 .PHONY: get-deps
 get-deps:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint
-	go install golang.org/x/tools/cmd/goimports
-	go install github.com/goreleaser/goreleaser
-	# For hyperparameter importance computation
-	go get github.com/ryanbressler/CloudForest
-	go install github.com/ryanbressler/CloudForest/growforest
-	go get github.com/vektra/mockery/v2/.../@v2.9.2
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.28.3
+	go install github.com/bufbuild/buf/cmd/buf@v0.42.1
+	go install golang.org/x/tools/cmd/goimports@v0.1.5
+	go install github.com/goreleaser/goreleaser@v1.1.0
+	go install github.com/ryanbressler/CloudForest/growforest@v0.0.0-20161201194407-d014dc32840a
```

Comment:
You mentioned pinning to latest to mitigate the problem with not having dependabot alerts. CloudForest should possibly be pinned to avoid unexpected changes since we bundle it into the product. Why not pin other build tools to latest?

Reply:
Just because drift may break CI without warning (golangci-lint definitely would). There may be a way to rework things such that pinning to latest is easier but it always could happen that tools make breaking changes and we need to update config files/scripts for them. I wanted to avoid that.